scripts/ci.sh

```diff
@@ -10,13 +10,3 @@ fi
 
 run_f "lerna bootstrap --no-ci"
 run_f "lerna run build"
-
-if [ "$1" == 'noGenerate' ]; then
-  run_f "lerna run test --ignore=generation"
-else
-  run_f "lerna run test"
```

Comment:
Unit tests run twice on Linux and Windows on Github actions. That should be enough, IMO.